From c2ef210039765c77544c6cd86336f4dc4b7ff52a Mon Sep 17 00:00:00 2001
From: Alexey Brodkin <abrodkin@synopsys.com>
Date: Wed, 16 Jan 2019 11:31:06 +0300
Subject: [PATCH] ARC: [hsdk]: Enable cgroups support

This is required for systemd, otherwise we get a message about
missing API filesystems.

Upstream status: Inappropriate.

Signed-off-by: Alexey Brodkin <abrodkin@synopsys.com>
---
 arch/arc/configs/hsdk_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arc/configs/hsdk_defconfig b/arch/arc/configs/hsdk_defconfig
index 109126c3d75c..4f1b917ba6f5 100644
--- a/arch/arc/configs/hsdk_defconfig
+++ b/arch/arc/configs/hsdk_defconfig
@@ -82,3 +82,4 @@ CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=10
 CONFIG_CRYPTO_ECHAINIV=y
 # CONFIG_ARC_SMP_HALT_ON_RESET is not set
 CONFIG_ARC_UBOOT_SUPPORT=y
+CONFIG_CGROUPS=y
-- 
2.16.2

